tests/test_cargo_alias_config.rs

```diff
@@ -0,0 +1,51 @@
+use support::{project, execs, basic_bin_manifest};
+use hamcrest::{assert_that};
+
+fn setup() {}
+test!(default_alias_config_overrides_config {
```

Comment:
I've tweaked the tests here to make the `test!` macro to go away, so this may need a rebase. You can just use `#[test]` now as usual